testeng/jobs/manageJenkinsJobs.groovy

```diff
@@ -0,0 +1,13 @@
+import jenkins.model.*
+
+// Ad-hoc job management script.
```

Comment:
NIT: 
I would call this Janitor or cleanup or something of the sort, just to use existing nomenclature, I would not type manage if I was trying to find this